uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/logging/formatter/SimpleTextFormatter.java

```diff
@@ -0,0 +1,40 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.client.ui.automation.logging.formatter;
+
+import android.util.Log;
+
+import com.microsoft.identity.client.ui.automation.logging.LogLevel;
+
+/**
+ * A log formatter that formats logs into a single string following the same format as used by the
+ * the Android {@link Log} utility.
```

Comment:
I don't understand this comment. Isn't this the [LogcatLikeFormatter](https://github.com/AzureAD/microsoft-authentication-library-common-for-android/blob/127a730282452d0477df4b01e110282bb8eca41c/uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/logging/formatter/LogcatLikeFormatter.java#L34-L38)?

Reply:
Yes, this is similar to LogcatLikeFormatter. 
But LogcatLikeFormatter converts the string into its own format.
While I wrote SimpleTextFormatter which returns the string as it is.

Reply:
Yeah so this javadoc is certainly wrong then and very misleading...we need a better javadoc here

Reply:
Please look at the new javadoc. Thanks

Reply:
Hi @shahzaibj ,
Please look at the resolution of this comment. Thanks.